table/block_based/filter_policy.cc

```diff
@@ -1446,12 +1471,12 @@ FilterBitsBuilder* BloomFilterPolicy::GetBuilderWithContext(
   }
 }
 
-const char* BloomFilterPolicy::kName() { return "bloomfilter"; }
+const char* BloomFilterPolicy::kClassName() { return "bloomfilter"; }
```

Comment:
Planning for the future:

For now, the canonical name needs to be "bloomfilter" so that an options file written by 7.0 can be read by many recent versions. But I'm sure we would like to move to another name like "rocksdb.BloomFilter" or something and let "bloomfilter" be the alternate name. So I think to get there, we can add an alternate name now for what we want as the future canonical name, and then in a year or so we can swap the canonical and alternate names.

(Same for ribbonfilter.)